wallet/src/main/scala/org/bitcoins/wallet/models/SpendingInfoDAO.scala

```diff
@@ -277,6 +277,15 @@ case class SpendingInfoDAO()(implicit
 
   }
 
+  /**
+    * Fetches all the incoming TXOs in our DB that are in
+    * the transaction with the given TXID
+    */
+  def findDbsForTx(txid: DoubleSha256DigestBE): Future[Vector[UTXORecord]] = {
```

Comment:
Same with this method, please add a unit test so we can detect regressions in the future. I think it would be good to also have a negative test case making sure we don't accidentally fetch outgoing txos (the scaladoc indicates this is just for incomign txos).

Reply:
Added test.

> I think it would be good to also have a negative test case making sure we don't accidentally fetch outgoing txos (the scaladoc indicates this is just for incomign txos).

All txos in the spendinginfodao is an incoming txo